.gitignore

```diff
@@ -28,3 +28,5 @@ next-env.d.ts
 
 # Cache
 .eslintcache
+
+__screenshots__/
```

Comment:
How is this being used?

Reply:
It seems the vitest will generate screenshot during running.